andela/ah-backend-sparta
# #164798218 Users should be able to bookmark and unbookmark articles

#### What does this PR do?

-  Authenticated users should be able to bookmark and un-bookmark 

#### Description of Task to be completed?
-  When a user is authenticated, they should be able to bookmark and unbookmark articles they could wish to read later on after some time.

#### How should this be manually tested?

- Fetch this branch `git fetch origin ft-bookmark-articles-164798218`
- Checkout to the branch `git checkout ft-bookmark-articles-164798218`
- Run `python manage.py migrate`
- Run `python manage.py runserver`
- The endpoint to bookmark an article but it should be a POST method ` {{base-url}}/api/articles/{{slug}}/bookmark/`
- The endpoint to unbookmark an article but it has to be a DELETE method ` {{base-url}}/api/articles/{{slug}}/bookmark/`
- The endpoint to get a user bookmarked articles ` {{base-url}}api/bookmarks/`
#### What are the relevant pivotal tracker stories?
[#164798218](https://www.pivotaltracker.com/story/show/164798218)

#### Postman Collection
[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/348bc66b2c0a9f651a3f)
